certified-connectors/N-able Cloud Commander/apiDefinition.swagger.json

```diff
@@ -0,0 +1,6000 @@
+                  "description": "Mail nickname of the group.",
+                  "title": "Mail nickname"
+                },
+                "onPremisesLastSyncDateTime": {
+                  "type": "string",
+                  "description": "Indicates when the group was synced to on premise.",
+                  "title": "On premise last sync"
+                },
+                "onPremisesSecurityIdentifier": {
+                  "type": "string",
+                  "description": "On premise security identifier of the group.",
+                  "title": "On premise security identifier"
+                },
+                "onPremisesSyncEnabled": {
+                  "type": "string",
+                  "description": "Indicates if on premise sync is enabled.",
+                  "title": "On premise sync enabled"
+                },
+                "preferredDataLocation": {
+                  "type": "string",
+                  "description": "Preferred data location of the group.",
+                  "title": "Preferred data location"
+                },
+                "proxyAddresses": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "description": "Proxy addresses of the group.",
+                    "title": "Proxy addresses"
+                  },
+                  "description": "Proxy addresses."
+                },
+                "renewedDateTime": {
+                  "type": "string",
+                  "description": "Renewed date time of the group.",
+                  "title": "Renewed"
+                },
+                "resourceBehaviorOptions": {
+                  "type": "array",
+                  "items": {
+                    "description": "Resource behavior options of the group.",
+                    "title": "Resource behaviour options",
+                    "type": "string"
+                  },
+                  "description": "Resource behavior options."
+                },
+                "resourceProvisioningOptions": {
+                  "type": "array",
+                  "items": {
+                    "description": "Resource provisioning options of the group.",
+                    "title": "Resource provisioning options",
+                    "type": "string"
+                  },
+                  "description": "Resource provsioning options."
+                },
+                "securityEnabled": {
+                  "type": "boolean",
+                  "description": "Indicates if the group is security enabled.",
+                  "title": "Security enabled",
+                  "enum": [
+                    true,
+                    false
+                  ]
+                },
+                "visibility": {
+                  "type": "string",
+                  "description": "Visibility of the group.",
+                  "title": "Visbility"
+                },
```

Comment:
Can you please correct the spelling "Visbility"

Reply:
Fixed